src/test/java/uk/gov/hmcts/divorce/caseworker/service/task/SetReIssueAndDueDateTest.java

```diff
@@ -57,4 +57,23 @@ void shouldSetDueDateAndReIssueDate() {
         assertThat(result.getData().getDueDate()).isEqualTo(expectedDueDate);
         assertThat(result.getData().getApplication().getReissueDate()).isEqualTo(expectedReIssueDate);
     }
+
+    @Test
+    void shouldNotChangeDueDateAndReIssueDateIfJointApplication() {
```

Comment:
shouldNotChangeDueDateAndReIssueDateIfJointApplication ? but we are changing reissue date ?